extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -24,11 +24,38 @@ limitations under the License.
 
 # amp-subscriptions
 
-## Introduction
+## Usage
 
 The `amp-subscriptions` extension implements subscription-style access/paywall rules.
 
-## Relationship to `amp-access`
+The solution comprises the following components:
+
+1. [**AMP Reader ID**][1]: provided by the AMP ecosystem, this is a unique identifier of the Reader as seen by AMP.
+2. [**Local service**][2]: provided by the Publisher to control and monitor access to documents.
+   1. [**Authorization endpoint**][3]: provided by the Publisher, returns the response that explains which part of a document the Reader can consume.
+   2. [**Login page**][4]: provided by the publisher, allows the Publisher to authenticate the Reader and connect their identity with AMP Reader ID.
+   3. [**Subscribe page**][5]: provided by the publisher, allows the Reader to purchase a subscription from the Publisher.
+   4. [**Pingback endpoint**][6]: provided by the Publisher, is used to send the “view” impression for a document.
+3. [**Vendor services**][7]: registered AMP extensions that cooperate with the main `amp-subscriptions` extension.
+4. [**Fallback entitlement**][8]: provided by the Publisher, determines what the Reader can see in the event that all services fail to respond to the calls made to the Authorization endpoints.
+5. [**Service score factors**][9]: provided by the Publisher, used if no service returns a valid entitlement to determine which service is used as the default service.
+6. [**Structured data markup**][10]: Schema.org page-level configuration, provided by the Publisher.
+7. [**Attributes**][11]: authored by the Publisher, defines which parts of a document are visible in which circumstances.
+8. [**Actions**][12]: fulfilled by a particular service to present the Reader with a specific experience.
+
+### Flow
+
+1. Google AMP Cache returns the document to the Reader with some sections obscured using [Attributes][11].
```

Comment:
What is the Reader? Is there supplementary documentation we could link to? How is is related to AMP Reader ID?

Reply:
This was detailed in the glossary that I copied over from the amp-access page: https://amp.dev/documentation/components/amp-access/#amp-glossary.